src/styles/_catalog.scss

```diff
@@ -1,22 +1,20 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  gap: 48px;
 
   &__card {
-    display: flex;
+    display: grid;
```

Comment:
Do you really need grid here?

Reply:
agree, `grid` for card is redundant, you can leave flex here